<launch>

  <node pkg="pose_cmds_logger" name="pose_cmds_logger" type="pose_cmds_logger.py" output="screen">
    <remap from="odom" to="icp_odom"/>
  </node>

</launch>
